### DISCUSSION TITLE
Is there a plan to train a Libritts data model?

### DISCUSSION THREAD

**Discussion by ronggong** [coqui-ai/TTS#403] [Ideas]
Hi,

I came from the TensorFlowTTS project, tried with their LibrTTS recipe of multispeakers model training. However, the results I got from their Fastspeech2 model is not plausible. It sounds metallic and buzzy. I wonder it is the dataset problem, e.g. hard to train, or it is the multispeaker problem. 

I wonder is there any plan in the project to having a Libristts model.

**Comment by erogol** [comment]
I want to train a model, but it is not in our immediate plans. However, if you like to do it, I'd be more than happy to help. 

Previously we trained some models with @Edresson and get good results but VCTK was easier to train and gave better results, so we did not release a LibriTTS model.

**Reply by skol101** [reply]
@erogol how about training VCTK and LibriTTS datasets on VITS model ? Do you think it might produce better results than VITS VCTK alone (which is already pretty good).

**Reply by erogol** [reply]
I've not tried libritts. It is a harder dataset but is worth to try. Maybe you can train vctk first then add libritts later on

**Comment by mortyzhou77** [comment]
> 

Hi, I guess Fastspeech2 and hifigan is sensitive for data quality and scale, when I finetuned the CSMSC single-speaker model (fs2+hifigan) by 3 hrs data from a single speaker, the inferenced speech sounds metallic and buzzy...
I guess its because libritts speech for one speaker is not enough? there are 2000+ speaker for just 500 hrs..
Do you find the reason why sounds metallic and buzzy on libritts? Thank you so much!

**Comment by chigkim** [comment]
Any estimate on how many hours it would take to train VITS on VCTK and LibriTTS using Nvidia a100 80gb?
